contracts/deposit-service/DepositServiceBase.sol

```diff
@@ -14,21 +14,23 @@ contract DepositServiceBase is IDepositServiceBase {
 
     // Using immutable storage to keep the constants in the bytecode
     address public immutable gateway;
+    address public immutable wrappedTokenAddress;
     bytes32 internal immutable wrappedSymbolBytes;
 
     constructor(address gateway_, string memory wrappedSymbol_) {
         if (gateway_ == address(0)) revert InvalidAddress();
 
         gateway = gateway_;
-
-        // Checking if token symbol exists in the gateway
-        if (IAxelarGateway(gateway_).tokenAddresses(wrappedSymbol_) == address(0)) revert InvalidSymbol();
-
+        wrappedTokenAddress = IAxelarGateway(gateway_).tokenAddresses(wrappedSymbol_);
         wrappedSymbolBytes = wrappedSymbol_.toBytes32();
+
+        // Wrapped token symbol param is optional
+        // When specified we are checking if token exists in the gateway
+        if (bytes(wrappedSymbol_).length > 0 && wrappedTokenAddress == address(0)) revert InvalidSymbol();
```

Comment:
```suggestion
        if (wrappedSymbol_.length == 0) return;

        wrappedTokenAddress = IAxelarGateway(gateway_).tokenAddresses(wrappedSymbol_);

        if (wrappedTokenAddress == address(0)) revert InvalidSymbol();

        wrappedSymbolBytes = wrappedSymbol_.toBytes32();
```

Reply:
@sammy1991106 in this PR we are allowing the empty `wrappedSymbol_ ` for the chains where we don't support their wrapped token (Aurora etc). So in this case `sendNative` and `unwrapNative` will be disabled. If funds are deposited anyway refund address can still refund